Mathlib/CategoryTheory/Comma/Over/Pullback.lean

```diff
@@ -156,14 +171,38 @@ def pullbackComp {X Y Z : C} (f : X ⟶ Y) (g : Y ⟶ Z) : pushout (f ≫ g) ≅
 instance pushoutIsLeftAdjoint {X Y : C} (f : X ⟶ Y) : (pushout f).IsLeftAdjoint  :=
   ⟨_, ⟨mapPushoutAdj f⟩⟩
 
+section HasInitial
+variable [HasInitial C]
+
 /-- If `X : C` is initial, then the under category of `X` is equivalent to `C`. -/
-def equivalenceOfIsInitial {C : Type*} [Category C] {X : C} (hX : IsInitial X) :
-    Under X ≌ C where
-  functor := Under.forget X
-  inverse := { obj Y := Under.mk (hX.to Y), map f := Under.homMk f }
-  unitIso := NatIso.ofComponents (fun Y ↦ Under.isoMk (Iso.refl _) (hX.hom_ext _ _))
-  counitIso := NatIso.ofComponents (fun _ ↦ Iso.refl _)
+@[simps!]
+noncomputable def forgetMapTerminal :
+    forget _ ≅ map (initial.to X) ⋙ (equivalenceOfIsInitial initialIsInitial).functor :=
+  NatIso.ofComponents fun X ↦ .refl _
+
+end HasInitial
+
+section HasBinaryCoproducts
+variable [HasBinaryCoproducts C]
+
+/-- The functor from `C` to `Under X` which sends `Y : C` to `in₁ : X ⟶ X ⨯ Y`,
+sometimes denoted `X*`. -/
+@[simps! obj_left obj_hom map_left]
+def star : C ⥤ Under X := Monad.free _ ⋙ algebraToUnder X
```

Comment:
```suggestion
/-- The functor from `C` to `Under X` which sends `Y : C` to `in₁ : X ⟶ X ⨿ Y`,
sometimes denoted `X*`. -/
@[simps! obj_left obj_hom map_left]
def star : C ⥤ Under X := Monad.free _ ⋙ algebraToUnder X
```
And I couldn't find the source of "sometimes denoted `X*`" for the existing `Over.star` to tell you whether the comment or the name is applicable here.

Reply:
Yeah I actually have no idea how this should be called

Reply:
Maybe remove the "sometimes denoted `X*`" comment but keep the name (or maybe `costar`?).